File: scripts/fizzbuzz.js
#!/usr/bin/env node
/**
 * Quick FizzBuzz: print 1..n (default 100). Run: node scripts/fizzbuzz.js [n]
 */
const max = Math.max(1, parseInt(process.argv[2] || "100", 10) || 100);
for (let n = 1; n <= max; n++) {
  const f = n % 3 === 0;
  const b = n % 5 === 0;
  if (f && b) console.log("FizzBuzz");
  else if (f) console.log("Fizz");
  else if (b) console.log("Buzz");
  else console.log(String(n));
}

File: scripts/fizzbuzz.rb
#!/usr/bin/env ruby
# Quick FizzBuzz: print 1..n (default 100). Run: ruby scripts/fizzbuzz.rb [n]

max = (ARGV[0] || 100).to_i
max = 1 if max < 1

(1..max).each do |n|
  puts(
    if (n % 15).zero? then "FizzBuzz"
    elsif (n % 3).zero? then "Fizz"
    elsif (n % 5).zero? then "Buzz"
    else n.to_s
    end
  )
end
